rpcd/playbooks/rpc-pre-upgrades.yml

```diff
@@ -19,18 +19,63 @@
   roles:
     - { role: "rpc_pre_upgrade", tags: [ "rpc-pre-upgrades" ] }
 
-- name: restart beaver
+- name: remove beaver
   hosts: all
   tags:
     - rpc-pre-upgrades
+    - remove-beaver
   tasks:
     - name: Check if beaver is installed
       stat:
         path: /etc/init.d/beaver
       register: beaver_init_script
+      changed_when: false
```

Comment:
https://github.com/ansible/ansible-modules-core/blob/stable-2.1/files/stat.py#L409

This module never returns changed=True, so this guard is unnecesary.